dist/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <link href="./assets/css/tailwind.css" rel="stylesheet">
```

Comment:
- [ ] There is a console error coming from this line. I believe that you already installed tailwind using postcss as instructed [here](https://tailwindcss.com/docs/installation#installing-tailwind-css-as-a-post-css-plugin)? If that's right, we don't need ths line. PostCSS already takes care of compiling tailwind styles.